### PR TITLE
Update history.txt

### PR DESCRIPTION
**Pull request by IllianiCBT**
Updated history.txt to include most recent batch of PRs

### CODE REVIEW

File: MekHQ/docs/history.txt [hunk 1]
```diff
@@ -38,6 +38,26 @@ MEKHQ VERSION HISTORY:
...
++ PR #4136: Updated Scenario Modifiers to More Consistently Contribute to Map Size
++ PR #4137: Tied StratCon Force Generation BV Allowance to AtB Difficulty Option
++ PR #4140: Added Dedicated Awards Panel to Personnel Tab of Campaign Options
++ PR #4141: Fixed Used Parts Value Multipliers Incorrectly Resetting to 0
++ PR #4142: Added No Commander Nag Dialog
++ PR #4143: Adjusted Founder Turnover Modifier & Updated Documentation for Turnover and Retention Module
++ PR #4144: Changed "Operational Victory Points" to "Scenario Victory Points"
++ PR #4145: Reversed Order of Personnel Logs, Renamed 'Personnel Log' to 'Personal Log'
++ PR #4146: Added Ability to Change Original Unit Assignment via Personnel Table
++ PR #4151: Updated Vehicle Gunner's Experience Level Calculation to Optionally Include Artillery Skill
```
Comment (Sleet01) [inline MekHQ/docs/history.txt:56]: Duplicate

**Review by Sleet01 — CHANGES_REQUESTED** [review]
Couple dupes and a line that could move up, otherwise looks good.

File: MekHQ/docs/history.txt [hunk 2]
```diff
@@ -38,6 +38,26 @@ MEKHQ VERSION HISTORY:
...
++ PR #4151: Updated Vehicle Gunner's Experience Level Calculation to Optionally Include Artillery Skill
++ PR #4150: Added Unit Quality Parameter to addNewUnit Method and Related Tests
++ PR #4153: Added Font Color Options to MHQ Options for Negative, Positive, and Warning Events
++ PR #4147: Set Default Edge Usage States to True
```
[the remark this answers is not in the document]
Comment (IllianiCBT) [reply MekHQ/docs/history.txt:60]: Ah, I thought they needed to be in order of merging. Easy fix.

[the remark this answers is not in the document]
Comment (Sleet01) [reply MekHQ/docs/history.txt:60]: Ignore this comment, I forgot that this was a roll-up of several consecutive merges over some time.

**Comment by IllianiCBT** [comment]
Resolved requested revisions

**Review by Sleet01 — APPROVED** [review]
LGTM

**Pull request merged**